fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -564,4 +565,30 @@ public boolean queryMetaGroupStable(long metaGroupId) {
         }
         return false; // return false if any error happens
     }
+
+    public List<Long> getWorkersByWorkerGroup(long workerGroupId) throws UserException {
+        List<Long> nodeIds = new ArrayList<>();
+        prepare();
+        try {
+            List<WorkerGroupDetailInfo> workerGroupDetailInfos = client.
+                    listWorkerGroup(serviceId, Collections.singletonList(workerGroupId), true);
+            for (WorkerGroupDetailInfo detailInfo : workerGroupDetailInfos) {
+                List<WorkerInfo> workerInfos = detailInfo.getWorkersInfoList();
+                for (WorkerInfo workerInfo : workerInfos) {
+                    if (workerToBackend.containsKey(workerInfo.getWorkerId())) {
+                        nodeIds.add(workerToBackend.get(workerInfo.getWorkerId()));
+                    } else {
+                        // for follower fe
```

Comment:
`workerToBackend` in leader may also be not complete.
better to update the map at the end.